docs/plugins/metadata

```diff
@@ -0,0 +1,75 @@
+# Metadata
+
+An App created with the SDK has a Metadata in it.
+
+Be aware that when your App is deployed to the App Store the Metadata is configured in the Store dashboard.
+
+## Usage
+
+Sometimes you may want to access Metadata in your App code directly. In those cases you can import the _Metadata plugin_ from the Lightning SDK.
```

Comment:
Perhaps it will make it more clear if we provide an example? Something like: "... in your App code directly, for example to display the App version in about screen."